guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -108,14 +108,14 @@ Before you begin the upgrade to {Project} 6.10, which includes an upgrade to Pul
 If you perform no migrations on 6.9, they will all be performed during the upgrade to 6.10.
 For smaller content setups, or if you can afford longer downtimes, you can proceed without the pre-migration.
 
-For {SmartProxies}, you can choose to re-deploy new 6.10 {SmartProxies} rather than to upgrade.
+For {SmartProxies}, you can choose to deploy new 6.10 {SmartProxies} rather than to upgrade.
 
 For future upgrades following {Project} 7.0, you will be required to upgrade the operating system from RHEL 7 to RHEL 8 on your {ProjectServer}s and {SmartProxies}.
 You can upgrade the operating system in-place or through a cloning process. The latter includes migration of all data, configuration, and synced content.
 
 [NOTE]
 ====
-If you are planning to avoid the upgrade from Pulp 2 to Pulp 3 and deploy a new {Project} 6.10 infrastructure due to the Pulp 3 changes instead, you may want to wait for {Project} 7.0 to deploy with RHEL 8 directly.
+If you are planning to avoid the upgrade from Pulp 2 to Pulp 3 and deploy a new {Project} 6.10 infrastructure due to the Pulp 3 changes instead, you might want to wait for {Project} 7.0 to deploy with RHEL 8 directly.
```

Comment:
This really should only have been applied to the 2.5 branch and not be merged to master. In master Pulp 2 is dead since master is really Satellite 7.0.

Reply:
I can close and make new PR.

Reply:
I don't mind either way. This may be a good PR to cherry pick to the 2.5 branch and then clean up master.

Reply:
So merge it and cherry-pick later. This are trivial style things after all.

Reply:
Indeed